xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/icons/default.iconset

```diff
@@ -1159,6 +1166,14 @@ xhtml_valid = xhtml_valid
 zoom = zoom
 zoom_in = zoom_in
 zoom_out = zoom_out
-filter = find
+
+## Deprecated since 16.3.0 to conform to the new naming convention.
```

Comment:
I guess the version should be changed

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2986/commits/2469e4d7dc0eb0353216c28cd8a1c2d34bbeab35 :+1: